pkgs/development/compilers/terra/default.nix

```diff
@@ -1,68 +1,87 @@
-{ lib, stdenv, fetchurl, fetchFromGitHub
-, llvmPackages, ncurses, lua
-}:
+{ lib, stdenv, fetchurl, fetchFromGitHub, llvmPackages, ncurses, cmake, libxml2
+, symlinkJoin, breakpointHook, cudaPackages, enableCUDA ? false }:
 
 let
-  luajitArchive = "LuaJIT-2.0.5.tar.gz";
+  luajitRev = "9143e86498436892cb4316550be4d45b68a61224";
+  luajitArchive = "LuaJIT-${luajitRev}.tar.gz";
   luajitSrc = fetchurl {
```

Comment:
```suggestion
  luajitSrc = fetchFromGitHub {
```

Reply:
Following SuperSandro2000 and vcunat advice, we're using  `fetchFromGitHub` instead `fetchurl`. See #32997.

Reply:
Got this fixed in [`c49d9ad` (#125896)](https://github.com/NixOS/nixpkgs/pull/125896/commits/c49d9adc385ccff57ef020456f3700fe6878619d).